### PR TITLE
Expand content on physical file upload in RP tutorial

### LINKED ISSUE #6020: Expand content on physical file upload in RP tutorial
https://docs.microsoft.com/aspnet/core/tutorials/razor-pages/uploading-files
*aspnetcore/tutorials/razor-pages/uploading-files.md*

The side note on physical file uploads doesn't quite add enough code to make it super clear what to do. I'm planning on expanding that code example a bit ... something along the lines of ...

```csharp
public async Task<IActionResult> OnPostAsync()
{
    if (!ModelState.IsValid)
    {
        return Page();
    }

    var filePath = "some path here";

    using (var fileStream = new FileStream(filePath, FileMode.Create))
    {
        await FileUpload.UploadPublicSchedule.CopyToAsync(fileStream);
    }

    return RedirectToPage("./Index");
}
```

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #6020 

@blowdart There are security comments in the warning about uploading malicious files that u should :eyes:. I don't see better content to link in our doc set. If you have alternate links, I'll put them in there.

### CODE REVIEW

File: aspnetcore/tutorials/razor-pages/uploading-files.md [hunk 1]
```diff
@@ -55,17 +55,36 @@ To avoid code duplication for processing uploaded schedule files, add a static h
 
 ### Save the file to disk
 
-The sample app saves the file's content into a database field. To save the file's content to disk, use a [FileStream](/dotnet/api/system.io.filestream):
+The sample app saves uploaded files into database fields. To save a file to disk or a network share, use a [FileStream](/dotnet/api/system.io.filestream). The following example copies a file held by `FileUpload.UploadPublicSchedule` to a `FileStream` in an `OnPostAsync` method. The `FileStream` writes the file to disk or a network share at the `<PATH-AND-FILE-NAME>` provided:
```
Comment (blowdart) [inline aspnetcore/tutorials/razor-pages/uploading-files.md]: Avoid network shares in an example, by default app pools are local accounts which can't access network shares. Instead maybe D:\uploadedFiles?

**Comment by blowdart** [comment]
There are no better links :) You could add something saying "Never persist user files into the same directory tree as your application."

**Comment by guardrex** [comment]
@blowdart Updates made ... and I added a note about what happens if you forget to put the file name on the path. Fortunately, I personally *never* make that mistake ... *NEVER!* ... at least not that our readers are aware of. 😬

**Comment by guardrex** [comment]
@blowdart Are the updates ok?

**Comment by blowdart** [comment]
Yup, LGTM

**Pull request merged**